Source/EasyNetQ/Producer/ClientCommandDispatcherSingleton.cs

```diff
@@ -30,6 +30,7 @@ public class ClientCommandDispatcherSingleton : IClientCommandDispatcher
                 StartDispatcherThread(configuration);
         }
 
+        /// <inheritdoc />
         public Task<T> InvokeAsync<T>(Func<IModel, T> channelAction, CancellationToken cancellationToken)
```

Comment:
Well, it seems that `cancellationToken` param is not used here. Maybe this is a mistake?

Reply:
Nope. We discussed this one with @micdenny in Slack: it seems that we should switch from background thread to async lock to utilize CancellationToken better. It will help to create multi-channel dispatcher as well.

Reply:
Does this mean that the token will be used in the future, but for now it does not affect anything?

Reply:
I pushed https://github.com/EasyNetQ/EasyNetQ/pull/1056/commits/634a184dabf530d0873846e4896b0c153f6068f4 which uses CancellationToken now.